projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -120,7 +120,7 @@
           </button>
           <a
             *ngSwitchDefault
-            href="{{ action.link }}"
+            href="{{ $any(action).link }}"
```

Comment:
don't forget this one as well 

is it possible to make isCardAction working for both? or we might have to define another function?

Reply:
I will remove `isCardAction(action)`, because it narrow `action` to `CardAction`. the `*ngSwitchDefault` is used for `CardLinkAction`. So, I need think about another solution.